code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -250,20 +250,6 @@
 	else if(bodytemperature > maxbodytemp)
 		adjustBruteLoss(heat_damage_per_tick)
 
-/mob/living/simple_animal/Bumped(AM as mob|obj)

```

Comment:
You mean when I want to push Besty or the uncooked bacon out of the way but she or it instead pops out two tiles behind the way I was pushing? Yes please, that shit is annoying